.github/workflows/tf_plan.yml

```
@@ -47,12 +48,22 @@ jobs:
 
       - name: Terraform Plan
         id: plan
-        run: terraform plan -no-color
+        run: terraform plan -no-color -input=false -out=tfplan
+
+      - name: Render plan diff
+        id: show
+        run: terraform show -no-color tfplan > tfplan.txt
+
+      - name: Render plan for PR comment
+        id: tf_output
+        run: |
+          echo 'PLAN_OUTPUT<<EOF' >> $GITHUB_OUTPUT
+          cat tfplan.txt >> $GITHUB_OUTPUT
+          echo 'EOF' >> $GITHUB_OUTPUT
```

This can take the alternative form
```suggestion
          {
            echo 'PLAN_OUTPUT<<EOF'
            cat tfplan.txt
            echo EOF
          } >> "$GITHUB_ENV"
```

Not sure which is more succinct or better supported in Actions. I'd been juggling them when trying to debug issues caused by TF output using the default `terraform_wrapper: true`. Both appear to result in the same output.